src/app/pages/apps/services/applications.service.ts

```diff
@@ -140,4 +143,47 @@ export class ApplicationsService {
   stopApplication(name: string): Observable<Job<ChartScaleResult>> {
     return this.ws.job('chart.release.scale', [name, { replica_count: 0 }]);
   }
+
+  convertDateToRelativeDate(date: Date): string {
+    const diff = Math.round(((new Date() as unknown as number) - (date as unknown as number)) / 1000);
```

Comment:
1. You may be able to use `formatRelative` from `date-fns` here.
2. Seems to be impossible to test this right now, but it looks like granularity is too much. It should be something like `Last 24 hours`, `Last 3 days`, `Last week`, `Last month`, not `5 hours ago`, `5 days ago`.

Reply:
@undsoft  - `date-fns` provides a strange solution with time, I was not able to find solution to fix it, 
If we can keep my method, I can remove unnecessary options and leave only few of them like:
`Last 24 hours, Last 3 days, Last week, Last month, Long time ago`

<img width="506" alt="Screenshot 2023-05-03 at 17 45 17" src="https://user-images.githubusercontent.com/22980553/235951495-67e0c9f1-674f-4afb-b47e-f922a445119c.png">

Reply:
Okay, let's do that.